src/remotecall.jl

```diff
@@ -768,6 +768,9 @@ close(rr::RemoteChannel) = call_on_owner(close_ref, rr)
 isopen_ref(rid) = isopen(lookup_ref(rid).c)
 isopen(rr::RemoteChannel) = call_on_owner(isopen_ref, rr)
 
+isempty_ref(rid) = isempty(lookup_ref(rid).c)
+Base.isempty(rr::RemoteChannel) = call_on_owner(isempty_ref, rr)
```

Comment:
```suggestion
isdone_ref(rid) = Base.isdone(lookup_ref(rid).c)
Base.isdone(rr::RemoteChannel) = call_on_owner(isdone_ref, rr)
```

`isempty` falls back to `isdone` if `isdone` is defined, so it is a slightly broader fix to use `isdone`.

Reply:
That doesn't seem to work, the MWE below hangs with only `Base.isdone()`.

Reply:
Right, because channels may be empty but not done.